src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -110,7 +110,7 @@ internal PackageVersion GetPackageVersionHeader(IPackageInfo packageInfo)
             var pkgResponse = this.client.ExecuteAndDeserializeWithContent<PackageVersion>(req);
             if (!pkgResponse.success)
             {
-                throw new Exception(pkgResponse.message);
+                throw new ApplicationException(pkgResponse.message);
```

Comment:
Please, no more System Exception thrown by user code :)

Reply:
@QilongTang I know this has been merged already, but as a future reference, `ApplicationException` is not recommended to be used anymore. From https://docs.microsoft.com/en-us/dotnet/api/system.applicationexception?view=netcore-3.1:
```
You should derive custom exceptions from the Exception class rather than the ApplicationException class. You should not throw an ApplicationException exception in your code, and you should not catch an ApplicationException exception unless you intend to re-throw the original exception.
```

I think in this particular case, the most appropriate fix would be to throw a `PackageManagerException` which derives from `Exception`.